### ISSUE TITLE
Next.js app router doesn't work on recent versions

### ISSUE THREAD

**Issue by tom-sherman** [stackblitz/webcontainer-core#1152]
### Describe the bug

A basic Next.js app directory doesn't work in Stackblitz

### Link to the blitz that caused the error

https://stackblitz.com/edit/stackblitz-starters-c2p2tt?file=app/page.jsx,README.md

### Steps to reproduce

1. `npm run dev` in the provided stackblitz

or

1. Start a new Next.js blitz
2. Upgrade to latest next.js version
3. Delete pages directory (if it exists) and add a simple page.jsx and layout.jsx

### Expected behavior

Works in dev mode

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Platform

- OS: macOS
- Browser: Arc (chromium)
- Version: 115.0.5790.170

### Additional context

Twitter thread: https://twitter.com/tomus_sherman/status/1688648122707087360

**Comment by d3lm** [comment]
Hey 👋 Thanks for reporting the issue here. We'll have a look at it and come back if there's any updates.

**Comment by remorses** [comment]
What was the issue?

**Comment by alvarlagerlof** [comment]
I tried forking the example. I see two issues:
1. Editing the text in page.tsx does not cause the live preview to refersh
2. Reloading the pages errors in the terminal with this: 
<img width="902" alt="image" src="https://github.com/stackblitz/webcontainer-core/assets/14835120/1b243ad0-4db7-4110-a947-ad2db31203b2">

**Comment by kirjavascript** [comment]
Thanks for testing! We'll take a look at why these are not working an post further updates here

**Comment by SamVerschueren** [comment]
I’m on my phone right now so can’t really test. I believe this should work now.

**Comment by alvarlagerlof** [comment]
Now I'm getting errors like this on all of my projects.

<img width="969" alt="image" src="https://github.com/stackblitz/webcontainer-core/assets/14835120/4d65c1a7-a259-4407-938b-f13a059c6680">

**Comment by remorses** [comment]
@alvarlagerlof it's a problem with edge runtime i think

**Comment by SamVerschueren** [comment]
Yes it's an issue with that package. It's on our radar to be investigated pretty soon.
